bedwars-plugin/pom.xml

```diff
@@ -230,6 +238,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
+                <version>3.3.0-SNAPSHOT</version>
```

Comment:
Heres why its not compiling

It doesnt seem like there is `3.3.0-SNAPSHOT`. It looks like the latest is `3.2.4`

Reply:
Plugin repository URL is missing. Default Maven plugin repository dosen't contain snapshot releases.

```xml
<pluginRepositories>
  <pluginRepository>
    <id>apache.snapshots</id>
    <url>https://repository.apache.org/snapshots/</url>
  </pluginRepository>
</pluginRepositories>
```

Reply:
It's a snapshot!